src/js/components/DataChart/DataChart.js

```diff
@@ -547,7 +547,8 @@ const DataChart = forwardRef(
         if (!property || (!horizontal && y) || (horizontal && !y)) {
           if (render) return render(value);
         } else {
-          const datum = data[axisValue];
+          // const datum = data[axisValue];
+          const datum = data[0];
```

Comment:
This change causes an issue with the axis labels. If we have the following code we end up with two xaxis labels that are the same value (Jun 23) coming from data[0].

```
const data = [
  { a: 1, b: 'one', c: 111111, d: '2020-06-24' },
  { a: 2, b: 'two', c: 222222, d: '2020-06-23' },
];

export const Simple = () => (
  <Box align="center" justify="start" pad="large">
    <DataChart
          data={data}
          series={['d', 'a']}
          axis={{
            x: { property: 'd' },
            y: { property: 'a' },
          }}
        />
  </Box>
);
```
<img width="445" alt="Screen Shot 2023-11-20 at 12 32 56 PM" src="https://github.com/grommet/grommet/assets/54560994/ffa2d6be-7f51-472b-86e6-c306834028ce">